authentik/root/settings.py

```diff
@@ -188,11 +188,16 @@ def j_print(event: str, log_level: str = "info", **kwargs):
     "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
 }
 
+REDIS_PROTOCOL_PREFIX = "redis://:"
+if bool(CONFIG.y("redis.tls")):
+    REDIS_PROTOCOL_PREFIX = "rediss://:"
+
 CACHES = {
     "default": {
         "BACKEND": "django_redis.cache.RedisCache",
         "LOCATION": (
-            f"redis://:{CONFIG.y('redis.password')}@{CONFIG.y('redis.host')}:"
+            REDIS_PROTOCOL_PREFIX
+            + f"{CONFIG.y('redis.password')}@{CONFIG.y('redis.host')}:"
```

Comment:
Please add `REDIS_PROTOCOL_PREFIX` to the f-string instead of concatenating it.

Reply:
Good catch.